shared/packages.sh

```diff
@@ -6,13 +6,13 @@ ZLIB_URL="https://zlib.net/$lib-$ver.tar.gz"
 ZLIB_DIR="$lib-$ver"
 
 lib=libpng
-ver=1.6.34
-LIBPNG_URL="https://ftp-osl.osuosl.org/pub/libpng/src/archive/xz/libpng16/$lib-$ver.tar.xz"
+ver=1.6.35
+LIBPNG_URL="https://download.sourceforge.net/libpng/$lib-$ver.tar.xz"
```

Comment:
I thought we wanted to not rely on sf.net anymore.

Reply:
forgot to comment on this: The ftp mirror is outdated and doesn't have that file ;)

Reply:
libpng 1.6.36 released in the meanwhile, but provides a performance gain for ARM, maybe is worth to get a bump.

Reply:
still sourceforge only and they removed the dead mirror completely 👎